src/DynamoCoreWpf/ViewModels/Core/GalleryViewModel.cs

```diff
@@ -104,6 +101,31 @@ public GalleryViewModel(DynamoViewModel dynamoViewModel)
         }
 
         /// <summary>
+        /// Set image path relative to the gallery directory. The method looks for the 
+        /// images alongside GalleryContents.xml. If a given image cannot be found, it 
+        /// searches under "Data" directory in the parent directory.
+        /// </summary>
+        /// <param name="galleryDirectory">
+        /// The directory in which the GalleryContents.xml resides
+        /// </param>
```

Comment:
If the image is not exist, the image won't be loaded (i.e. only text will be loaded)